core-java-modules/core-java-lang-math-4/src/test/java/com/baeldung/percentile/CalculatePercentileUnitTest.java

```diff
@@ -0,0 +1,75 @@
+package com.baeldung.percentile;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+import java.util.stream.LongStream;
+
+import org.junit.jupiter.api.Test;
+
+public class CalculatePercentileUnitTest {
```

Comment:
Let's show the 0th and 100th percentiles in the test too just as a boundary sanity check

Reply:
tests added